pymc/sampling.py

```diff
@@ -1393,6 +1397,27 @@ def _choose_backend(trace: Optional[Union[BaseTrace, List[str]]], **kwds) -> Bac
     return NDArray(vars=trace, **kwds)
 
 
+def _init_trace(
```

Comment:
Does this need a test?

Reply:
It's covered extensively by the existing tests, though I agree that after extraction it is now easier to test it separately.
But my plan is to refactor this into using McBackend. This `_init_trace()` will then become `run.init_chain()` and then I'd also invest in the tests.